frontend/src/metabase/browse/utils.ts

```diff
@@ -0,0 +1,47 @@
+import type { SearchResult } from "metabase-types/api";
+
+export type SortableModel = Partial<
+  Pick<SearchResult, "id" | "name" | "collection">
+>;
+
+/** Sort models firstly by collection name (case insensitive), secondly by collection id, thirdly by model name (case insensitive). */
+export const sortModels = (a: SortableModel, b: SortableModel) => {
```

Comment:
since we're now grouping by collection, we don't need to sort by collection anymore, right?